example/declarative.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+library grinder.example.declarative;
+
+import 'package:grinder/grinder.dart';
+
+const main = grind;
```

Comment:
Yeah, that'll make it easier to tweak the example code to add programmatic tasks too.

Done.